utils/database.go

```diff
@@ -20,10 +20,11 @@ func NewDatabase(logger *zap.Logger, env *environment.Env) *mongo.Database {
 		env.Get(environment.MongoHost),
 		env.Get(environment.MongoDatabase))
 
-	logger.Info(fmt.Sprint(env.Get(environment.MongoUser),
-		env.Get(environment.MongoPassword),
-		env.Get(environment.MongoHost),
-		env.Get(environment.MongoDatabase)))
+	logger.Info("db connection details",
+		zap.String("user", env.Get(environment.MongoUser)),
+		zap.String("password env var name", environment.MongoPassword),
```

Comment:
I don't think we should be logging the database password

Reply:
We're not, it's logging the name of the env var that stores the database password

Reply:
Ah, that's fine